enterprise-suite/gotests/testenv/testenv.go

```diff
@@ -41,25 +55,51 @@ func InitEnv() {
 		Expect(err).To(Succeed(), "new k8sclient")
 	}
 
+	var additionalArgs []string
+	if isMinikube {
+		additionalArgs = append(additionalArgs, "--set exposeServices=NodePort")
+	}
+	if isOpenshift {
```

Comment:
I added a check for `default` and `gp2` storage classes. Seems a bit strange to use any storage class that exists, in case cluster admins set up something not meant for us...